mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -469,4 +471,13 @@ def test_read_vmrk_annotations():
     except FileNotFoundError:
         pass
 
+    raw = read_raw_brainvision(vhdr_path)
+    events, event_id = events_from_annotations(raw)
+    for key, values in event_id.items():
+        assert int(key[-3:].lstrip()) == values
+
+    assert _bv_parser("Stimulus/S 11") == 11
```

Comment:
I would rather like to see a battery of falling edge cases than the expected.

what would happen if I get something like `'stiMOulus/S'`, `'Stimulus\S'`, `'foobar/09876564578'` ?

Reply:
This is not how the Brainvision analyser works (... in principle one could set up pycorder to work like that I guess? But to do that one needs to be sufficiently python savy to quickly come up with a parser)

Again, I'm fine with suggestions for alternatives, but it must get in by .18.

Reply:
I mean, that we should make sure that the else branch of this line https://github.com/mne-tools/mne-python/pull/6276/files#diff-db95532aad4fae1b22f34cbf50ed7f19R845 does not generate unexpected behaviors.

Reply:
As far as I know the recorder, these events will never be interesting for epoching; i.e., it's always NewSegment or something like that.

Right @cbrnr ?

Reply:
Yes, these are not important for epoching, but we still need to catch them.

Reply:
So I'm +1 for adding failing test cases as @massich suggests. Like I said, I think we should return all events not just the ones relevant for epoching (users can and should filter out events relevant for them after we return everything, which will be easy because they can filter by value, i.e. between 0-255 and 1000-1255 if they only want stimulus and response markers).